gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/NullValue.java

```diff
@@ -62,7 +63,7 @@ public NullValue build() {
     }
 
     @Override
-    public Builder set(Void value) {
+    public Builder set(com.google.protobuf.NullValue value) {
```

Comment:
To override the function (since I set the BaseMarshaller from <Void, NullValue, Builder> to <com.google.protobuf.NullValue, NullValue, Builder>), I think this has to be the function signature.  I could also not override this function (since it now takes a value) and make changes elsewhere.  What's your opinion?